test/XRP/signer-test.ts

```diff
@@ -99,19 +99,19 @@ describe('signer', function(): void {
     const wallet = new FakeWallet(fakeSignature)
     const destinationAddress = Utils.decodeXAddress(
       'XVPcpSm47b1CZkf5AkKM9a84dQHe3m4sBhsrA4XtnBECTAc',
-    )
+    )!
     const sourceAddress = Utils.decodeXAddress(
       'X7vjQVCddnQ7GCESYnYR3EdpzbcoAMbPw7s2xv8YQs94tv4',
-    )
+    )!
```

Comment:
Never seen ! at the end of a const assignment, what does this operator do?

Reply:
This unwraps an optional T to a T. Specifically, it is transforming a  `WalletGenerationResult | undefined` to `WalletGenerationResult`. 

Aside: sorry these changes slipped in. Reverted.